src/reaction_lammps_mupt/reaction_template_builder/reaction_template_pipeline/util.py

```
@@ -0,0 +1,179 @@
+    }
+
+    return formatted_dict , data_smiles_list
+
+def prep_for_3d_molecule_generation(data_smiles_list, molecule_dict_csv_path_dict):
+    molecule_dict = {}
+    for i, smiles in enumerate(data_smiles_list):
+        key = f"data_{i+1}"
+        molecule = Chem.MolFromSmiles(smiles)
+        molecule = Chem.AddHs(molecule)
+        molecule_dict[key] = molecule
+    for i, (key, value) in enumerate(molecule_dict_csv_path_dict.items()):
```

Ruff (0.14.14)

[warning] Loop control variable `key` not used within loop body

(B007)